server/routes/app.js

```diff
@@ -41,6 +41,8 @@ async function getApp(req, res) {
         config.get('ldapAuthEnabled') || config.get('enableLdapAuth'),
       oidcConfigured: config.oidcConfigured(),
       oidcLinkHtml: config.get('oidcLinkHtml'),
+      showServiceTokensUI:
+        config.get('serviceTokenSecret') || config.get('serviceTokenSecret_d'),
```

Comment:
Can you wrap this with `Boolean(config.get('serviceTokenSecret') || config.get('serviceTokenSecret_d'))`. Secret would leak to UI otherwise

Reply:
@rickbergfalk done! nice catch